backend/app/README.md

```diff
@@ -1,5 +1,82 @@
+2. Encode the PLR trust certificate in base64 format to be used as input to build docker image if you are running the application in Docker.
+3. Install the Docker Desktop and start the Docker service.
+<br><br>
+**Note:** When you run the application, or follow the instructions for running it in Docker, a database named phlat, with
+the user phlat, and a host/container named phlat_postgres will be created. These details are required to connect to the
+database using the pgAdmin client, which is accessible from another container at http://localhost:5050/. 
+
+## Running using the IDE
+1. For running through your favorite editor Visual Studio Code or IntelliJ, you can run the application by running the main class `BackendApplication`.
+2. Set the above environment variables in the IDE run configuration.
+3. The run command will start Postgres DB container and Postgres Admin container which can be opened on [http://localhost:5050](http://localhost:5050).
+4. Adjust the path to the `compose.yaml` in the `application-local.yml` file if IDE complains about the path to the `compose.yaml` file.
+   ```yaml
+   file: "./backend/app/compose.yaml"
+   ```
+5. The Backend application will be available at [http://localhost:8088/](http://localhost:8088/) and swagger documentation at [http://localhost:8088/swagger-ui/index.html#](http://localhost:8088/swagger-ui/index.html#).
+
+## Running the Application using Spring Boot Maven Plugin
+1. Make sure the required environment variables are set in the command window where you are running the application.
+2. Adjust the path to the `compose.yaml` in the `application-local.yml` file if the command complains about the path to the `compose.yaml` file if the plugin cannot find it.
+   ```yaml
+   file: "./compose.yaml"
+   ```
+3. Switch to `<project root>/backend/app` dir and execute the following command to run the application:
+   ```sh
+   mvn spring-boot:run
+   ```
+4. The command will start Postgres DB container and Postgres Admin container which can be opened on [http://localhost:5050](http://localhost:5050).
+5. The Backend application will be available at [http://localhost:8088/](http://localhost:8088/) and swagger documentation at [http://localhost:8088/swagger-ui/index.html#](http://localhost:8088/swagger-ui/index.html#).
+
+## Using Docker
+It is recommended to run the application in Docker locally also to avoid any issues in cloud deployment, 
+especially when any configuration related to Docker or application properties are changed.
+
+### Steps
+1. Start the postgres DB and Postgres Admin containers if not running already using the following command.<br>
+This command will also create a network if not existing already, to communicate between the containers. 
+The network name follows the format: "folder/project name where the compose file is located"_common-network (in our case it should be app_common-network)
```

Comment:
For some reason it is called app_default on my Linux machine.

pluzhni@utux:~/work/plr/phlat/moh-phlat/backend/app$ docker network ls
NETWORK ID     NAME              DRIVER    SCOPE
1a4805261a8b   app_default       bridge    local
03e931a0969e   bridge            bridge    local
6aca9a1ad03d   compose_default   bridge    local
c930a2e10dfc   host              host      local
0137c3fd10ef   none              null      local

Might need to mention it.

Reply:
That was me. I forgot to rebase before trying it again.